qt/scientific_interfaces/Indirect/ISISCalibration.cpp

```diff
@@ -365,20 +371,23 @@ void ISISCalibration::setDefaultInstDetails() {
   try {
     setDefaultInstDetails(getInstrumentDetails());
   } catch (std::exception const &ex) {
-    g_log.warning(ex.what());
+    showMessageBox(ex.what());
```

Comment:
Does showMessageBox also post to the logger? I would recommend always having a persistent log of errors in case a user closes a box too early